sympy/integrals/tests/test_integrals.py

```diff
@@ -881,6 +881,20 @@ def test_as_sum_trapezoid():
     assert Integral(sign(x), (x, 0, 1)).as_sum(1, 'trapezoid') == S.Half
 
 
+def test_as_sum_simpson():
+    e = Integral(sqrt(x), (x, 0, 2))
+    k = Dummy('k', integer=True, positive=True)
+    assert e.as_sum(2, method="simpson", evaluate=False).equals(
+            sqrt(2)/3 + \
+            4*Sum(sqrt(2*k - 1), (k, 1, 1))/3 + \
+            2*Sum(sqrt(2)*sqrt(k), (k, 1, 0))/3)
+    assert e.as_sum(4, method="simpson", evaluate=False).equals(
+            sqrt(2)/6 + \
+            Sum(sqrt(k), (k, 1, 1))/3 + \
+            2*Sum(sqrt(k - S.Half), (k, 1, 2))/3)
```

Comment:
It can be better to evaluating the expression to closed form, like the test cases written above than using adhoc numeric tests like `equals`.